### PR TITLE
test: Have a longer timeout in all tests in kubelib

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

**Comment by stefwalter** [comment]
Aim to fix this:

```
# testDashboard (check_kubernetes.TestKubernetes)
#
bash: /etc/kubernetes/start-kubernetes: No such file or directory
Restarting browser...
Restarting browser...
> [object Object]
> [object Object]
> [object Object]
Wrote TestKubernetes-testDashboard-FAIL.png
Journal extracted to TestKubernetes-testDashboard-10.111.120.223-FAIL.log
not ok 50 testDashboard (check_kubernetes.TestKubernetes) duration: 257s
Traceback (most recent call last):
  File "/build/cockpit/test/verify/kubelib.py", line 430, in testDashboard
    self.check_logs(b)
  File "/build/cockpit/test/verify/kubelib.py", line 294, in check_logs
    b.wait_in_text("#content .containers-listing tbody.open tr.listing-ct-item td:last-child", "running")
  File "/build/cockpit/test/common/testlib.py", line 241, in wait_in_text
    return self.wait_js_func('ph_in_text', selector, text)
  File "/build/cockpit/test/common/testlib.py", line 211, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "/build/cockpit/test/common/testlib.py", line 688, in <lambda>
    return lambda *args: self._invoke(name, *args)
  File "/build/cockpit/test/common/testlib.py", line 711, in _invoke
    raise Error(res['error'])
Error: timeout
```

**Comment by dperpeet** [comment]
Wouldn't it be nicer to just do that in `kubelib.KubernetesCase.setUp`? The storage tests in `storagelib.py` do something similar, even if not with timeouts.

**Comment by stefwalter** [comment]
Sure, can try that.

File: test/verify/kubelib.py [hunk 1]
```diff
@@ -614,13 +615,13 @@ def testNodes(self):
     def testTopology(self):
         m = self.machine
         b = self.browser
+        b.wait_timeout(120)
```
Comment (dperpeet) [inline test/verify/kubelib.py]: removed for merge

**Pull request closed without merging**